keyboards/m65/m65.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "quantum.h"
+
+#define KEYMAP( \
```

Comment:
```suggestion
#define LAYOUT( \
```

And any references to the old name